benchmarks/python/test_transformer.py

```diff
@@ -44,381 +44,293 @@
 
 
 def create_transformer_forward(fd: FusionDefinition) -> None:
```

Comment:
nit:
```Suggestion
def transformer_forward_fusion(fd: FusionDefinition) -> None:

Reply:
Is this just for consistency with other files? I usually make function names start with a verb.

Reply:
Yes, I try to maintain a pattern in case we need to use pattern matching for any feature later.
Can be ignored for now.